esctest/tests/xterm_winops.py

```diff
@@ -12,288 +16,599 @@
 # 9;0 - Restore maximized window
 
 class XtermWinopsTests(object):
+  def delay_after_icon(self):
```

Comment:
Functions follow a naming convention of CamelCase, so this should be `DelayAfterIcon`. The only exception is tests, which much begin with `test_` to be recognized as tests.

Reply:
ok